### PR TITLE
feat: grid

### PR DESCRIPTION
**Pull request by shivangisharma01**
image
image

### CODE REVIEW

File: components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    align-items: center;
+    width: 14rem;
+    height: 4.2rem;
+    background-color: var(--hold-color);
+    color: var(--color-white);
+
+}
+  
+
+  .content-grid-two {
```
Comment (shashidwivedi) [inline components/Grid/grid.css]: You can name .content-grid-two as .grid-two-cols and move it above .grid-items ruleset.
Similarly, for .content-grid-three as .grid-three-cols

**Review by shashidwivedi — APPROVED** [review]
The changes are just suggestions. You can take care of it later.

File: components/Grid/grid.css (hunk 1 above)
Comment (shivangisharma01) [reply components/Grid/grid.css]: okay

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean can merge

**Pull request merged**